platform/platform.go

```diff
@@ -111,7 +111,8 @@ type Options struct {
 
 // RuntimeConfig contains cluster-specific configuration.
 type RuntimeConfig struct {
-	OutputDir string
+	OutputDir    string
+	Distribution string
```

Comment:
By the existing schema, this should be in `Options` instead of `RuntimeConfig` because it doesn't vary between clusters in a run.  Is there a reason it should be here instead?

Reply:
Ah yeah, `Options` makes sense. I'll switch it.